platooning_control/src/platoon_control_worker.cpp

```diff
@@ -16,36 +16,23 @@
 
 ------------------------------------------------------------------------------*/
 
-#include "platoon_control_worker.h"
+#include "platoon_control/platoon_control_worker.hpp"
 
 
 namespace platoon_control
 {
-   
-	PlatoonControlWorker::PlatoonControlWorker()
+
+    PlatoonControlWorker::PlatoonControlWorker()
     {
         pid_ctrl_ = PIDController();
-        pp_ = PurePursuit();
-
-    }
 
-    void PlatoonControlWorker::updateConfigParams(PlatooningControlPluginConfig new_config)
-    {
-        ctrl_config_ = new_config;
-        pid_ctrl_.config_ = new_config;
-        pp_.config_ = new_config;
     }
 
-	double PlatoonControlWorker::getLastSpeedCommand() const {
+    double PlatoonControlWorker::getLastSpeedCommand() const {
         return speedCmd_;
     }
 
-    void PlatoonControlWorker::setCurrentPose(const geometry_msgs::PoseStamped msg)
-	{
-		current_pose_ = msg.pose;
-	}
```

Comment:
do we not need this?

Reply:
Since we are using a shared_ptr now, its currently being set here https://github.com/usdot-fhwa-stol/carma-platform/blob/7377dea870ee1c0d87c570f015c74d7d7502d536/platooning_control/src/platoon_control.cpp#L64

No strong opinions on this though, i can add this method back

Reply:
Updated, current_pose_ is no longer required in platoon_worker. Its only being used for steering_calculation which is handled by pure_pursuit